Libs/DICOM/Widgets/ctkDICOMAppWidget.cpp

```diff
@@ -77,7 +79,9 @@ class ctkDICOMAppWidgetPrivate: public Ui_ctkDICOMAppWidget
   ~ctkDICOMAppWidgetPrivate();
 
   ctkFileDialog* ImportDialog;
+#if QT_VERSION >= QT_VERSION_CHECK(5,0,0)
```

Comment:
The check should be done based on enabled/disabled features. We should avoid features disappearing silently based on detected Qt version.